src/stores/beslutter-store.ts

```diff
@@ -3,5 +3,14 @@ import createUseContext from 'constate';
 
 export const useBeslutterStore = createUseContext(() => {
 	const [beslutterProsessStartet, setBeslutterProsessStartet] = useState(false);
-	return { beslutterProsessStartet, setBeslutterProsessStartet };
+	const [harBeslutter, setHarBeslutter] = useState(false);
+	const [godkjentAvBeslutter, setGodkjentAvBeslutter] = useState(false);
```

Comment:
Trenger ikke å ting i storen som kan utledes fra data som vi allerede har. (F.eks i vedtaket som vi henter fra backend)
Har vurdert om vi egentlig trenger storen.  `beslutterProsessStartet` og `setBeslutterProsessStartet` ble lagt til for å slippe å gjøre unødvendige kall til backend, men kan hende at vi ønsker å gjøre det likevel for å få mindre kompleksitet i frontend.